docs/context/ref/Action.md

```diff
@@ -50,17 +63,30 @@ A context object with which the action will be performed
 
 </details>
 
+<details>
+  <summary><code>channelId</code></summary>
+
+**type**: `string`
+
+Broadcasts a context on this channel, if the `broadcast` action has been defined.
+The `channel` property is ignored unless the `action` is broadcast.
```

Comment:
Same again here - do use this version as I think the second line is important to include.